corehq/apps/app_manager/tests/test_apps.py

```diff
@@ -275,17 +276,23 @@ def testBuildTemplateApps(self, mock):
     @patch('corehq.apps.app_manager.models.validate_xform', return_value=None)
     def testGetLatestBuild(self, mock):
         factory = AppFactory(build_version='2.40.0')
-        factory.new_basic_module('register', 'case', with_form=False)
-        factory.app.save()
+        m0, f0 = factory.new_basic_module('register', 'case')
+        f0.source = BLANK_TEMPLATE.format(xmlns=f0.unique_id)
```

Comment:
Any reason this shouldn't be part of `new_basic_module`?

Reply:
It causes a lot of test failures

Reply:
fair enough